server/admin/api.go

```diff
@@ -56,6 +56,86 @@ func (s *Server) apiConfig(w http.ResponseWriter, _ *http.Request) {
 	writeJSON(w, s.core.ConfigMsg())
 }
 
+func (s *Server) apiAsset(w http.ResponseWriter, r *http.Request) {
+	assetSymbol := strings.ToLower(chi.URLParam(r, assetSymKey))
+	assetID, found := dex.BipSymbolID(assetSymbol)
+	if !found {
+		http.Error(w, fmt.Sprintf("unknown asset %q", assetSymbol), http.StatusBadRequest)
+		return
+	}
+	asset, err := s.core.Asset(assetID)
+	if err != nil {
+		http.Error(w, fmt.Sprintf("unsupported asset %q / %d", assetSymbol, assetID), http.StatusBadRequest)
+		return
+	}
+
+	var errs []string
+	dexAsset := asset.Asset
+	backend := asset.Backend
+	currentFeeRate, err := backend.FeeRate()
+	if err != nil {
+		errs = append(errs, fmt.Sprintf("unable to get current fee rate: %v", err))
+	}
+
+	synced, err := backend.Synced()
+	if err != nil {
+		errs = append(errs, fmt.Sprintf("unable to get sync status: %v", err))
+	}
+
+	res := &AssetInfo{
+		Asset:          dexAsset,
+		CurrentFeeRate: currentFeeRate,
+		Synced:         synced,
+		Errors:         errs,
+	}
+	writeJSON(w, res)
+}
+
+func (s *Server) apiAssetPOST(w http.ResponseWriter, r *http.Request) {
+	assetSymbol := strings.ToLower(chi.URLParam(r, assetSymKey))
+	assetID, found := dex.BipSymbolID(assetSymbol)
+	if !found {
+		http.Error(w, fmt.Sprintf("unknown asset %q", assetSymbol), http.StatusBadRequest)
+		return
+	}
+	asset, err := s.core.Asset(assetID)
+	if err != nil {
+		http.Error(w, fmt.Sprintf("unsupported asset %q / %d", assetSymbol, assetID), http.StatusBadRequest)
+		return
+	}
+
+	body, err := ioutil.ReadAll(r.Body)
+	r.Body.Close()
+	if err != nil {
+		http.Error(w, fmt.Sprintf("unable to read request body: %v", err), http.StatusInternalServerError)
+		return
+	}
+	if len(body) == 0 {
+		http.Error(w, "no POST data provided", http.StatusBadRequest)
+		return
+	}
+
+	var data AssetPost
+	if err = json.Unmarshal(body, &data); err != nil {
+		http.Error(w, fmt.Sprintf("invalid POST data: %v", err), http.StatusBadRequest)
+		return
+	}
+
+	var settingsFound bool
+
+	if data.FeeRateScale != nil {
+		settingsFound = true
+		log.Infof("Setting %s (%d) fee rate scale factor to %f", strings.ToUpper(assetSymbol), assetID, *data.FeeRateScale)
+		asset.Backend.SetFeeRateScale(*data.FeeRateScale)
```

Comment:
Seems like your just using the backend as a `float64` field here. The `Backend` doesn't really need to know anything about this scale factor, does it?

Reply:
I don't follow.  The backend applies the scale to the result of estimatesmartfee internally:  https://github.com/decred/dcrdex/pull/864/files/479ef31c6924215cb9fa09134553f4f878bfb6b9#diff-51dc1aa092977ea75e0a2e2e2bc5263254edd21ca3f653b0a27035c5c667c829R378-R386

Reply:
But every `Backend` is expected to apply the factor in exactly the same way, which to me indicates that it's not really a `Backend` role. The fee adjustment is just not the concern of the `Backend`. This factor can just as easily be applied by the caller without saddling asset developers with boilerplate.

Reply:
Right, gotcha.  I did it this way as a lazy way of making it asset-specific (and perhaps applicable to other rate uses even though there are none), but that is indeed achievable with a scale map in Swapper where it gets applied in `(*Swapper).Negotiate`